packages/schemas/src/player/gamemodes/general/quests/modes/arenabrawl.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import { APIData, removeFormatting } from "@statsify/util";
+import { Field } from "../../../../../metadata";
+import { FormattedGame } from "../../../../../game";
+import { GameQuests } from "../game-quests";
+import { QuestTime, getAmountDuring, questFieldData } from "../util";
+import { add } from "@statsify/math";
+
+export class ArenaQuests implements GameQuests {
+  @Field({ ...questFieldData, leaderboard: { name: "Play Arena (Daily)" } })
+  public dailyPlayArena: number;
+
+  @Field(questFieldData)
+  public kills: number;
+
+  @Field(questFieldData)
+  public wins: number;
+
+  @Field({ ...questFieldData, leaderboard: { name: "Play Arena (Weekly)" } })
```

Comment:
The leaderboard metadata won't get spread properly here because you overwrite the leaderboard field.